src/components/breadcrumb/Breadcrumb.js

```diff
@@ -10,53 +11,75 @@ import messages from './messages';
 
 import './Breadcrumb.scss';
 
-const constructChildren = (children, threshold) => {
-    const dotDotDotItems = children.slice(1, children.length + 1 - threshold);
-    const menuCrumbsItems = (
-        <EllipsisCrumb>
-            {dotDotDotItems.reverse().map((crumb, index) => (
-                <MenuLinkItem key={index}>{crumb}</MenuLinkItem>
-            ))}
-        </EllipsisCrumb>
-    );
-    return [...children.slice(0, 1), menuCrumbsItems, ...children.slice(1 - threshold)];
-};
-
-const renderBreadcrumbs = (children, threshold) => {
-    let newChildren = children;
-    let { length } = children;
-    let hasEllipsis = false;
-
-    if (length > threshold) {
-        newChildren = constructChildren(children, threshold);
-        length = newChildren.length;
-        hasEllipsis = true;
-    }
-
-    return React.Children.map(newChildren, (item, i) => {
-        const isLastCrumb = length === 0 || i === length - 1;
-        return (
-            <Crumb className={hasEllipsis && i === 1 ? 'no-shrink' : undefined} isLastCrumb={isLastCrumb}>
-                {item}
-            </Crumb>
-        );
-    });
-};
-
-type Props = {
+export type BreadcrumbProps = {
     /** Class name for Breadcrumb */
     children: React.Node,
     /** Breadcrumb contents */
     className?: string,
+    /** Show number of breadcrumb items before overflow. Default is 1 */
+    itemsBeforeOverflow?: number,
```

Comment:
Is this the number of crumb items that are displayed before the Ellipsis crumb?

I.e. in this screenshot, does `items` refer to the house icon?
<img width="382" alt="Screen Shot 2022-08-31 at 6 56 50 PM" src="https://user-images.githubusercontent.com/7311041/187815973-5a9590c7-0eae-43aa-8a33-6683f9ff06d5.png">

Reply:
Yes, the house icon is the number of items

Reply:
Is there a way we could name this so that it's more obvious this is a number? Without reading the description, it sounds like it could be a list of components

Reply:
maybe something like `[...]Count`?

Reply:
i renamed numItemsBeforeOverflow